katas/content/complex_arithmetic/complex_addition/Solution.qs

```diff
@@ -0,0 +1,16 @@
+namespace Kata {    
+     open Microsoft.Quantum.Math;
+     
+    operation ComplexAdd(x : Complex, y: Complex) : Complex {
+        
+        let a = x::Real;
+        let b = x::Imag;
+
+        let c = y::Real;
+        let d = y::Imag;
+    
+        let z = Complex(a + c, b + d);
+
+        return z;
```

Comment:
We can make the code a lot more concise using unwrap operator and returning the result right away without a temp variable

```suggestion
        let (a, b) = x!;
        let (c, d) = y!;

        return Complex(a + b, c + d);
```

The same approach should be used in the later tasks